spec/defines/sensu_handler_spec.rb

```diff
@@ -177,4 +177,9 @@
     it { should contain_sensu_handler('myhandler').with_handle_flapping( true ) }
   end
 
+  context 'handle_silenced' do
```

Comment:
I've now also added a spec to check when `handle_silenced` is explicitly set to `false`